src/difference/solution-class-sunday/difference.spec.js

```diff
@@ -0,0 +1,70 @@
+import { difference } from './difference.js';
```

Comment:
And i think we should add tests for returning new array and original one is not modified